lua/neo-tree/utils.lua

```diff
@@ -317,7 +317,7 @@ M.open_file = function(state, path, open_cmd)
       return
     end
     if state.current_position == "current" then
-      vim.cmd(open_cmd .. " " .. escaped_path)
+      pcall(vim.cmd, open_cmd .. " " .. escaped_path)
```

Comment:
You can't just swallow errors without informing the user. Is there a particular error that could happen where the file is actually opened properly but vim outputs a useless error message anyhow? If there is, you need to handle that one specific case and then re-throw or log.error(...) if there was any other reason for the error.